### PR TITLE
feat: import view data properties

### PR DESCRIPTION
**Pull request by niekvanstaveren**

### CODE REVIEW

File: src/qae/index.ts [hunk 1]
```diff
@@ -0,0 +1,10 @@
+import { exportProperties, importProperties } from "../conversion";
+import data from "./data";
+import properties from "./object-properties";
+
+export default () => ({
```
Comment (haxxmaxx) [inline src/qae/index.ts:5]: does this need to be a function?

File: src/conversion/import-properties.ts [hunk 2]
```diff
@@ -81,6 +89,11 @@ const importProperties = (
     setValue(propertyTree, "qProperty.qHyperCubeDef.qMeasures", measures);
   }
 
+  if (viewDataMode) {
+    setValue(propertyTree, "qProperty.totals.show", false);
```
Comment (haxxmaxx) [inline src/conversion/import-properties.ts:93]: if `totals: {}` doesn't exist, is it created?

File: src/ext/index.js [hunk 3]
```diff
@@ -34,7 +34,15 @@ export default function ext(env) {
...
       cssScaling: true,
     },
-    importProperties,
-    exportProperties,
+    importProperties(exportFormat, initialProperties, extension, hypercubePath) {
+      const defaultPropertyValues = {
+        defaultDimension: extension.getDefaultDimensionProperties(),
+        defaultMeasure: extension.getDefaultMeasureProperties(),
+      };
+      return importProperties({ exportFormat, initialProperties, extension, hypercubePath, defaultPropertyValues });
```
Comment (haxxmaxx) [inline src/ext/index.js:42]: I'm looking at the `importProperties` call in the `convertTo` method in nebula. The parameters don't really match. Is it a different `importProperties` being called in nebula?

File: src/conversion/import-properties.ts (hunk 2 above)
Comment (niekvanstaveren) [reply src/conversion/import-properties.ts:93]: I would assume so yes. https://github.com/qlik-trial/conversion/blob/6f6141c69506e2c2afdccf8497471679d686fea4/src/utils/index.js#L43-L64

File: src/qae/index.ts (hunk 1 above)
Comment (niekvanstaveren) [reply src/qae/index.ts:5]: Nope :)

File: src/conversion/import-properties.ts (hunk 2 above)
Comment (haxxmaxx) [reply src/conversion/import-properties.ts:93]: great

**Review by haxxmaxx — APPROVED** [review]

**Pull request merged**